services/booking-api/lambdas/getHistoricalAttendees.js

```diff
@@ -0,0 +1,39 @@
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  console.log('EVENT: ', JSON.stringify(event, undefined));
+
+  let referenceCode = event.pathParameters?.referenceCode;
+
+  if (!referenceCode) {
```

Comment:
This kind of neg-checking is a bit hard to spot. Maybe the check could be a bit more verbose?

Reply:
The neg-checking is actually one of the better methods to use if you want to check for falsy values like undefined, false, null and empty string at the same time. Otherwise you could end up with multiple checks like:

```js
if (referenceCode === undefined | referenceCode === '' | referenceCode === false. ...) { ... }
```

Another solution would be to default referenceCode to a falsy value like:

```js
let referenceCode = event.pathParameters?.referenceCode || '';

if (referenceCode === '') { ... }
```